modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/select_organization_role.jspf

```diff
@@ -19,13 +19,20 @@ List<Organization> organizations = null;
 String organizationIds = ParamUtil.getString(request, "organizationIds");
 
 if (step == 1) {
-	organizations = OrganizationLocalServiceUtil.getOrganizations(StringUtil.split(organizationIds, 0L));
+	String keywords = ParamUtil.getString(request, "keywords");
+
+	if (Validator.isNull(keywords)) {
+		organizations = OrganizationLocalServiceUtil.getOrganizations(StringUtil.split(organizationIds, 0L));
+	}
+	else {
+		organizations = OrganizationLocalServiceUtil.search(selUser.getCompanyId(), -1, keywords, null, null, null, null, QueryUtil.ALL_POS, QueryUtil.ALL_POS);
```

Comment:
can we use `OrganizationConstants.ANY_PARENT_ORGANIZATION_ID` instead of -1 as second param?